setup.cfg

```diff
@@ -38,8 +38,10 @@ audio =
 vision =
     numpy>=1.18.0
     opencv-python>=4.5.0
+    ffmpeg-python>=0.2.0
```

Comment:
Is this just a small wrapper for ffmpeg? Can you install this if you don't have ffmpeg available?